src/pages/signUpView.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import TextInput from '../components/global/TextInput';
+import Button from '../components/global/Button';
+import { Link } from "react-router-dom";
+import SignUpBGImage from '../assets/images/SignupImage';
+
+export default function SignUpView( ) {
+    const [ firstName, setFirstName ] = useState("");
+    const [ lastName, setLastName ] = useState("");
+    const [ email, setEmail ] = useState("");
+    const [ password, setPassword ] = useState("");
+    const [ firstNameInfoText, setFirstNameInfoText ] = useState("");
+    const [ lastNameInfoText, setLastNameInfoText ] = useState("");
+    const [ emailInfoText, setEmailInfoText ] = useState("");
+    const [ passwordInfoText, setPasswordInfoText ] = useState("");
+
+    const signUp = () => {
+        // For debugging
+        console.log(`First name: ${firstName} \nLast name: ${lastName} \nEmail: ${email} \nPassword: ${password}`);
+
+
+        // Check inputs
+        if (!firstName) {
+            setFirstNameInfoText("Please enter your first name");
+        }
+        if (!lastName) {
+            setLastNameInfoText("Please enter your last name");
+        }
+        if (!email) {
+            setEmailInfoText("Please enter an email");
+        }
+        if (!password) {
+            setPasswordInfoText("Please enter a password");
+        }
+
+
+        // Reset variables
+        setFirstName("");
+        setLastName("");
+        setEmail("");
+        setPassword("");
+    }
+
+    return (
+        <div className="sign-up--root">
+            <div className="sign-up--container">
```

Comment:
Hmm, I wonder if it's actually better to use a Form here as convention

Reply:
I thought about it, but after doing some research it doesn't seem that forms are that necessary anymore. Their main use is if you're sending the data straight to a server